frontend/components/Button/index.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import Text from 'root/components/Text';
+
+import './index.css';
+
+const Button = ({ handleClick, href }) => {
+  const Component = href ? 'a' : 'button';
+
+  return (
+    <Component styleName="root" type="submit" href={href} onClick={handleClick}>
```

Comment:
I don't know... we should not have `href` in a button component and the same thing applies to the `type` in `a` tags, we should make this explicit or remove `type` in `a` tags and `href` in buttons.

Reply:
Buttons and links should have the same styles either, but they do, so I'm treating them as they were the same to make my work easier.

What good would it be to have 2 components to do the exact same thing? Yet, I agree type shouldn't be there

Reply:
It's ok, merge. I think react omits `null` attributes from the HTML output.